frontend/src/pages/ForgotPassword.vue

```diff
@@ -5,7 +5,9 @@
         <div class="header-body text-center mb-7">
           <b-row class="justify-content-center">
             <b-col xl="5" lg="6" md="8" class="px-2">
+              <!-- eslint-disable-next-line @intlify/vue-i18n/no-dynamic-keys-->
               <h1>{{ $t(displaySetup.headline) }}</h1>
+              <!-- eslint-disable-next-line @intlify/vue-i18n/no-dynamic-keys-->
```

Comment:
You deleted those keys. I suggest to grep this lint instruction to find more keys that are generated by code.